session/session.go

```diff
@@ -1068,6 +1069,19 @@ func CreateSession(store kv.Storage) (Session, error) {
 	return s, nil
 }
 
+func loadSystemTZ(se *session) (string, error) {
+	sql := `select variable_value from mysql.tidb where variable_name = "system_tz"`
+	rss, errLoad := se.Execute(context.Background(), sql)
+	defer rss[0].Close()
+	if errLoad != nil {
+		return "", errLoad
+	}
+	chk := rss[0].NewChunk()
+	defer chk.Reset()
```

Comment:
Why reset it?

Reply:
Per comment, chk need to be closed. I could not found any Close method binding to chk.

Reply:
comment from other reviewers. Not the comment of the function.